<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDatasetDialog</class>
 <widget class="QDialog" name="ExportDatasetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="layoutTop">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabDataset">
        <attribute name="title">
         <string>Dataset</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Dataset name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editName"/>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkTimestamp">
           <property name="text">
            <string>Include timestamp</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Render size:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboRender">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Scale size:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboScale"/>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelInput">
           <property name="text">
            <string>Input images:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="labelInputValue">
           <property name="text">
            <string>45</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelPer">
           <property name="text">
            <string>Per image:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labelCycles">
           <property name="text">
            <string>Cycles:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="labelOutput">
           <property name="text">
            <string>Output images:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="labelOutputValue">
           <property name="text">
            <string>22500</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="spinPerImage">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="spinCycles">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabParameters">
        <attribute name="title">
         <string>Parameters</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelPan">
           <property name="text">
            <string>Pan:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="hlPan">
           <item>
            <widget class="QLineEdit" name="editPanLow">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editPanHigh">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelTilt">
           <property name="text">
            <string>Tilt:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="hlTilt">
           <item>
            <widget class="QLineEdit" name="editTiltLow">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editTiltHigh">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelRoll">
           <property name="text">
            <string>Roll:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="hlRoll">
           <item>
            <widget class="QLineEdit" name="editRollLow">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editRollHigh">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelFOV">
           <property name="text">
            <string>Field of view:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="hlFOV">
           <item>
            <widget class="QLineEdit" name="editFOVLow">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editFOVHigh">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="checkDistortion">
           <property name="text">
            <string>Apply lens distortion</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>K1:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="hlK1">
           <item>
            <widget class="QLineEdit" name="editK1Low">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editK1High">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Noise of K2:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="hlK2Noise">
           <item>
            <widget class="QLineEdit" name="editK2NosieVar">
             <property name="text">
              <string>-10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="9" column="1">
          <widget class="QCheckBox" name="checkLabels">
           <property name="text">
            <string>Render label images</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="1">
          <widget class="QCheckBox" name="checkColorAdjust">
           <property name="text">
            <string>Adjust colors</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabImages">
        <attribute name="title">
         <string>Images</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="layoutSelection">
           <item>
            <widget class="QPushButton" name="buttonAll">
             <property name="text">
              <string>All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonNone">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListView" name="listImages"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="layoutBottom">
     <item>
      <layout class="QHBoxLayout" name="layoutProgress">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="labelElapsed">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRemaining">
         <property name="text">
          <string>13:27:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="buttonAbort">
         <property name="text">
          <string>Abort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonStart">
         <property name="text">
          <string>Start Export</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
